dev/NavigationView/NavigationView.xaml

```diff
@@ -811,6 +811,7 @@
         <Setter Property="MinHeight" Value="0"/>
         <Setter Property="HorizontalContentAlignment" Value="Stretch"/>
         <Setter Property="AutomationProperties.AccessibilityView" Value="Raw"/>
+        <Setter Property="Foreground" Value="{ThemeResource NavigationViewItemHeaderForeground}"/>
```

Comment:
nit: move the Foreground setter property to the top much like the ordering for NavigationViewItem